apps/web/package.json

```diff
@@ -21,6 +21,8 @@
     "axe-core": "^4.9.1",
     "cypress-axe": "^1.5.0",
     "html-react-parser": "^5.1.10",
+    "mobx": "^6.12.3",
+    "mobx-react-lite": "^4.0.7",
```

Comment:
This repo doesn't look like it's being maintained. Should we explore an alternative? 
https://github.com/mobxjs/mobx-react-lite

Reply:
Yeah that's the old package. The one we're getting comes from here:

https://github.com/mobxjs/mobx/tree/main/packages/mobx-react-lite

They moved it to a package of the main mobx.
https://github.com/mobxjs/mobx-react-lite#-this-repo-has-been-moved-to-mobx

Currently 100 on Snyk!
https://snyk.io/advisor/npm-package/mobx